fix: overlapping text and image

angeloanan opened the pull request:
This fix uses flexbox (display flex) to solve the overlap happening on small screens.

Preview after the fix
![After Image](https://user-images.githubusercontent.com/2129163/91273473-83804e00-e7a7-11ea-9934-f44614032834.png)

Resolves #10 

Signed-off-by: Christopher Angelo <angelo@angeloanan.xyz>

angeloanan committed:
fix: overlapping text and image
This fix uses flexbox (display flex)

Signed-off-by: Christopher Angelo <angelo@angeloanan.xyz>

DuckMasterAl reviewed (changes requested):
Do you think you can fix this separation of the "Learn More" button? Also, can 
you add some padding to the text since the learn more button makes it very cramped as you can see below.
<img width="361" alt="Screen Shot 2020-08-26 at 10 57 00 AM" src="https://user-images.githubusercontent.com/43557963/91371869-32d91700-e7e0-11ea-9154-dea67a7bfc3e.png">

angeloanan committed:
fix: button wrap
Having an `a` tag as a button is not big brain.
This commit changes the `a` tag to a `button` tag

Signed-off-by: Christopher Angelo <angelo@angeloanan.xyz>

DuckMasterAl reviewed (changes requested):
~~nevermind sorry~~

DuckMasterAl reviewed (approved):
Awesome but now I need to deal with conflicts... ~~fun~~

DuckMasterAl committed:
Fixing Conflicts :tada:
